api.md

```diff
@@ -1041,8 +1046,11 @@ DROP VIEW <view_name>
 #### Examples [](continuous_aggregate-drop-examples)
 Drop existing continuous aggregate.
 ```sql
-DROP VIEW contagg_view;
+DROP VIEW contagg_view CASCADE;
 ```
+>:TIP: `CASCADE` will drop those objects that depend on the continuous
```

Comment:
should this be a warning instead since it can potentially drop user objects

Reply:
That makes sense.

Reply:
Fixed.